PublishTestResultScreenshotV0/index.ts

```diff
@@ -37,26 +45,76 @@ async function run() {
 
 run();
 
-async function uploadScreenshots(failedTests: ShallowTestCaseResult[]) {
+/**
+ * Given system [build or release] returning the Id of the latest TestRun 
+ * @param system 
+ * @returns Latest TestRun Id
+ */
+async function getTestRunIdBy(system : string): Promise<number> {
+    let ret :number = -1;
+    let testRuns : TestRun[] = [];
+    let today = new Date();
+    let yesterday = new Date();
```

Comment:
This could return a lot of runs, I don't expect a run to take more than an hour, perhaps we could scope this down to a 2 hours or worst case 4 hours timeframe to limit the results returned instead of 24hours.

Reply:
You are right. A shorter query time range will work better. 2 hrs is a good suggestion, will add it  to the PR later.